compiler/rustc_ast_lowering/src/lib.rs

```diff
@@ -617,7 +624,7 @@ impl<'a, 'hir> LoweringContext<'a, 'hir> {
         let num_nodes = self.item_local_id_counter.as_usize();
         let (nodes, parenting) = index::index_hir(self.tcx, node, &bodies, num_nodes);
         let nodes = hir::OwnerNodes { opt_hash_including_bodies, nodes, bodies };
-        let attrs = hir::AttributeMap { map: attrs, opt_hash: attrs_hash };
+        let attrs = hir::AttributeMap { map: attrs, opt_hash: attrs_hash, define_opaques };
```

Comment:
why part of the `AttributeMap` instead of the `OwnerInfo`? :thinking:

Reply:
Yea this is a small incremental perf regression: https://github.com/rust-lang/rust/pull/138500#issuecomment-2725581054